shared/settings/advanced/index.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import * as React from 'react'
+import {globalStyles, globalMargins} from '../../styles'
+import {Box, Button, Text} from '../../common-adapters'
+
+type Props = {
+  onDBNuke: () => void,
+}
+
+function DBNuke(props: Props) {
+  return (
+    <Box
+      style={{
+        ...globalStyles.flexBoxColumn,
+        alignItems: 'center',
+        flex: 1,
+        padding: globalMargins.medium,
+        paddingTop: globalMargins.xlarge,
```

Comment:
Copied the styling from the desktop "Delete Me" flow. Looks ok to me on mobile, but let me know if I should change it.